### DISCUSSION TITLE
Condition between two object data fields

### DISCUSSION THREAD

**Discussion by FloLog9** [pimcore/pimcore#8512] [General]
Hi ! 
I want to show a textarea field depending on the value of the select field.  
I try with JS in "postOpenObject" listener but I can't select the correct dom because the objectbrick who contains these fields load after my JS. 

![image](https://user-images.githubusercontent.com/46871723/111809785-a43ffa80-88d5-11eb-9890-b343f0130597.png)

The fields concerned are "Phase" and "Détails de fin de vie". 

Thanks

**Comment by ckemptner** [comment]
Maybe something useful here? https://talk.pimcore.org/t/dynamic-select-box-multiselect-box-based-on-other-field/3186/7

**Reply by FloLog9** [reply]
Thanks, I will take a look
